app/lib/account/backend.dart

```diff
@@ -86,7 +86,12 @@ UserSessionData? get userSessionData =>
 /// When no associated User entry exists in Datastore, this method will create
 /// a new one. When the authenticated email of the user changes, the email
 /// field will be updated to the latest one.
-Future<AuthenticatedUser> requireAuthenticatedUser(
+Future<AuthenticatedUser> requireAuthenticatedUser() async {
+  return await _requireAuthenticatedUser(
+      expectedAudience: activeConfiguration.pubSiteAudience);
+}
+
+Future<AuthenticatedUser> _requireAuthenticatedUser(
```

Comment:
I guess next step is renaming this `requiredAuthenticatedAgent` :partying_face:

Reply:
rename + restructure